### DISCUSSION TITLE
Feat : custom scrollbar

### DISCUSSION THREAD

**Discussion by rahilansari261** [ArslanYM/StarterHive#237] [Ideas]
Hello @ArslanYM @masif2002 
As this issue https://github.com/ArslanYM/StarterHive/issues/233 metioned the default scroll bar not looking good.
so i am trying to make a custom one.

and i already added a custom scrollbar in https://github.com/ArslanYM/StarterHive/pull/236 here.
images are attatched below in light and dark mode.

### **Before** in light mode
![Screenshot from 2023-06-10 10-52-33](https://github.com/ArslanYM/StarterHive/assets/12849916/1cecda12-ab8d-4fdf-b790-a00f1748b6d9)

### **Before** in dark mode
![Screenshot from 2023-06-10 10-52-53](https://github.com/ArslanYM/StarterHive/assets/12849916/56087fc6-c8ee-4b40-b2c4-c5bd258449bf)

### **After** in light mode
![Screenshot from 2023-06-10 10-53-21](https://github.com/ArslanYM/StarterHive/assets/12849916/6875caab-9dc8-4930-8141-73a1beb9da15)

### **After** in dark mode
![Screenshot from 2023-06-10 10-53-43](https://github.com/ArslanYM/StarterHive/assets/12849916/2be14178-ec8f-44ec-9353-8ef4934350f2)

i need your suggestions.

**Comment by masif2002** [comment]
@rahilansari261 Great work 🚀 

Here are my suggestions:
1. The horizontal scrollbar looks fat. I think you could reduce the height
2. I would prefer the track for the vertical scrollbar to be transparent (instead of light grey). If you make this change, also make sure to change the color of the scrollbar in dark mode to something bright

**Comment by rahilansari261** [comment]
> @rahilansari261 Great work 🚀 
> 
> Here are my suggestions:
> 1. The horizontal scrollbar looks fat. I think you could reduce the height
> 2. I would prefer the track for the vertical scrollbar to be transparent (instead of light grey). If you make this change, also make sure to change the color of the scrollbar in dark mode to something bright
> 
> 

Noted 👍🏼
@ArslanYM anything you wanna add?

**Reply by ArslanYM** [reply]
Keep the scroll bar width low as well. That would be all.

**Comment by rahilansari261** [comment]
struggling with dark mode since it is created with react context provider not from the tailwind suggests i think i need more time to fix it
any help or suggestions would be appreciated 🙏🏼

**Reply by ArslanYM** [reply]
Whats the issue youre facing?

**Reply by rahilansari261** [reply]
Vertical scroll bar is not working in dark mode i added the background colour to transparent but it shows the white colour
You can see here


 ![244849500-2be14178-ec8f-44ec-9353-8ef4934350f2.png](https://github.com/ArslanYM/StarterHive/assets/12849916/57a85c7f-78d2-4605-a83f-dc7b6edace93)

**Reply by rahilansari261** [reply]
> Vertical scroll bar is not working in dark mode i added the background colour to transparent but it shows the white colour
> You can see here
> 
> 
>  ![244849500-2be14178-ec8f-44ec-9353-8ef4934350f2.png](https://github.com/ArslanYM/StarterHive/assets/12849916/57a85c7f-78d2-4605-a83f-dc7b6edace93)
> 
> 

Horizontal scroll bar is working fine now in both modes

**Reply by masif2002** [reply]
> Vertical scroll bar is not working in dark mode i added the background colour to transparent but it shows the white colour You can see here

If transparent is not working, can we make the background of the scrollbar to be black in dark mode?

**Reply by rahilansari261** [reply]
I tried that but i didn't get access to the dark class

**Reply by rahilansari261** [reply]
@masif2002 can you take this issue
I'm busy these days as well

**Reply by masif2002** [reply]
Sure.. Please assign me @ArslanYM

**Reply by ArslanYM** [reply]
You can start working on it brother.
